src/OrbitGl/ThreadTrack.cpp

```diff
@@ -120,31 +120,41 @@ std::string ThreadTrack::GetBoxTooltip(const Batcher& batcher, PickingId id) con
   const InstrumentedFunction* func =
       capture_data_->GetInstrumentedFunctionById(timer_info->function_id());
 
-  std::string function_name;
+  std::string label;
   bool is_manual = timer_info->type() == TimerInfo::kApiScope;
 
   if (func == nullptr && !is_manual) {
     return GetTimesliceText(*timer_info);
   }
 
   if (is_manual) {
-    function_name = timer_info->api_scope_name();
+    label = timer_info->api_scope_name();
   } else {
-    function_name = func->function_name();
+    label = func->function_name();
   }
 
-  std::string module_name =
-      func != nullptr
-          ? orbit_client_data::function_utils::GetLoadedModuleNameByPath(func->file_path())
-          : "unknown";
+  std::string module_name = orbit_client_data::CaptureData::kUnknownFunctionOrModuleName;
+  std::string function_name = orbit_client_data::CaptureData::kUnknownFunctionOrModuleName;
+  if (func != nullptr) {
+    module_name = orbit_client_data::function_utils::GetLoadedModuleNameByPath(func->file_path());
+    function_name = label;
+  } else if (timer_info->address_in_function() != 0) {
+    function_name = capture_data_->GetFunctionNameByAddress(timer_info->address_in_function());
+
+    const auto* module = capture_data_->FindModuleByAddress(timer_info->address_in_function());
+    if (module != nullptr) {
+      module_name = module->name();
+    }
```

Comment:
nit. assign module_name before function_name to match above code ordering?

Reply:
Sure.